src/app/swap/swap-out.ts

```diff
@@ -1,52 +1,48 @@
-import { BTC_NETWORK, getColdStorageConfig, getSwapConfig } from "@config"
+import { BTC_NETWORK, getSwapConfig } from "@config"
 import { TxDecoder } from "@domain/bitcoin/onchain"
 import { SwapServiceError } from "@domain/swap/errors"
 import { OnChainService } from "@services/lnd/onchain-service"
 import { toSats } from "@domain/bitcoin"
 import { SwapOutChecker } from "@domain/swap"
 import { baseLogger } from "@services/logger"
 import { SwapService } from "@services/swap"
-import { getActiveLnd } from "@services/lnd/utils"
-import { getChannelBalance } from "lightning"
 import { addAttributesToCurrentSpan } from "@services/tracing"
+import { getOffChainChannelBalances } from "@app/lightning"
 
 const logger = baseLogger.child({ module: "swap" })
 
-export const swapOut = async (
-  amount: Satoshis,
-): Promise<SwapOutResult | SwapServiceError | null> => {
+export const swapOut = async (): Promise<SwapOutResult | SwapServiceError | null> => {
```

Comment:
we are trying to avoid using null in our codename (there is still many of them from the time we didn't have this rules). we took that from Typescript codebase

Reply:
fixed with NoSwapAction type https://github.com/GaloyMoney/galoy/pull/1396/commits/203a2497f63b8df3f0e60b86c014182b54073831